usl_models/usl_models/flood_ml/dataset.py

```diff
@@ -16,6 +16,91 @@
 from usl_models.flood_ml import model
 
 
+def load_dataset(
+    sim_names: list[str],
+    batch_size: int = 4,
+    n_flood_maps: int = constants.N_FLOOD_MAPS,
+    m_rainfall: int = constants.M_RAINFALL,
+    max_chunks: Optional[int] = None,
+    firestore_client: Optional[firestore.Client] = None,
+    storage_client: Optional[storage.Client] = None,
+) -> tf.data.Dataset:
+    """Creates a dataset which generates chunks for flood model inference.
+
+    For training with teacher-forcing, `load_dataset_windowed` should be used instead.
```

Comment:
I think it'd be helpful to describe when to use this function, as opposed to when not to use it as we do here.